src/20_Components/amp-addthis.html

```diff
@@ -0,0 +1,84 @@
+<!---{
+  "disablePlayground": true
+}--->
+<!--
+  ## Introduction
+  AddThis Floating Bar and Inline Share Buttons are available for AMP! Make it simple for website visitors to share your content with our free social sharing buttons. Allow your visitors to connect directly to 200+ social networks, including Facebook, Twitter, WhatsApp, and more.
+
+  AddThis is trusted by over 15,000,000 websites with over 2 billion unique users, sharing content all over the world, in more than sixty languages.
```

Comment:
Is it OK for you to remove this sentence? AMP by Example is tech focused and this is very marketing heavy.

Reply:
I will remove this sentence and push an update shortly.